app/src/main/java/com/heapixLearn/discovery/logic/contact/contacts_domain_interfaces/DBStoreContact.java

```diff
@@ -0,0 +1,12 @@
+package com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces;
+
+public interface DBStoreContact {
```

Comment:
@dianaprimshits unused interface, should be deleted

Reply:
@YodaMike I don't think that this interface is unused.
If I delete it, logic will have to get contacts from the database. Perhaps, in our case, your comment is valid only for ServerContact and UIContact, because DBContact contains all Information, that we need to build Logic Contact. 
Correct me, if I'm wrong.